x-pack/test/observability_ai_assistant_functional/tests/feature_controls/assistant_security.spec.ts

```diff
@@ -131,9 +132,16 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await testSubjects.missingOrFail(ui.pages.links.globalHeaderButton);
       });
       it('shows no AI Assistant conversations link in global search', async () => {
+        await PageObjects.common.navigateToUrl('home', '', {
+          ensureCurrentUrl: true,
+          shouldLoginIfPrompted: false,
+        });
         await PageObjects.navigationalSearch.searchFor('observability ai assistant');
         const results = await PageObjects.navigationalSearch.getDisplayedResults();
-        expect(results.length).to.eql(0);
+        const aiAssistantConversationsEntry = results.find(
```

Comment:
Use `some` if you are interest whether it is any or none.
```suggestion
        const aiAssistantConversationsEntry = results.some(
```

Reply:
Thanks, I changed this